lib/bypass.ex

```diff
@@ -44,7 +44,7 @@ defmodule Bypass do
     pid = start_instance(opts)
     port = Bypass.Instance.call(pid, :port)
     debug_log("Did open connection #{inspect(pid)} on port #{inspect(port)}")
-    bypass = %Bypass{pid: pid, port: port}
+    bypass = %Bypass{pid: pid, port: port, endpoint_url: "http://localhost:#{port}/"}
```

Comment:
`localhost` is usually resolves to loopback interface address  - 127.0.0.1, however it is possible to bind to different interface. Therefore, the URL might not work.

It's safer to use use ` :inet.sockname/1` https://www.erlang.org/doc/man/inet.html#sockname-1 to retrieve listening address and then derive an interface address from `:inet.getifaddrs/0` https://www.erlang.org/doc/man/inet.html#getifaddrs-0

Reply:
I see some overlap with a PR I just created. For the ip address I made this PR: https://github.com/PSPDFKit-labs/bypass/pull/137